### PR TITLE
Tiles, Navbar, Footer, Responsiveness

### PR DESCRIPTION
**Pull request by UladBohdan**
- proper author aligning. We were talking about that, now they are on the same distance from the bottom.
screencapture-localhost-3001-main-page-main-page-html-1515458300769

### CODE REVIEW

**Comment by UladBohdan** [comment]
Let's keep discussing!

@tyndria something was wrong with title color/boldness/font, how should we try to adjust it?

File: src/tiles/tile.ejs [hunk 1]
```diff
@@ -10,7 +10,7 @@
 <% var titleVar; %>
 
 <div class="tile is-parent">
-    <article class="card tile is-child">
+    <article class="card tile is-child" style="display: flex;">
```
Comment (tyndria) [inline src/tiles/tile.ejs]: Let's do not use inline styles

File: src/tiles/tile.scss [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    align-self: flex-end;
+
+    .author {
+      align-self: flex-end;
+      font-family: $tile-primary-text;
+      margin-left: 5px;
+      margin-top: 10px;
+      width: 100%;
+
+      img {
```
Comment (tyndria) [inline src/tiles/tile.scss]: Do not repeat css code, please ('img' and 'name' classes)

**Comment by tyndria** [comment]
@UladBohdan , I think, we need to wait for latest Vika's PR, because she has redesign tiles

File: src/tiles/tile.ejs (hunk 1 above)
Comment (UladBohdan) [reply src/tiles/tile.ejs]: The problem is `tile` class sets `display` property to `block` and adding the property to `article` does not help. I can introduce  the 4th class for this tag, does that sounds better?

File: src/tiles/tile.scss (hunk 2 above)
Comment (UladBohdan) [reply src/tiles/tile.scss]: Which repetitions are you talking about?..

**Comment by UladBohdan** [comment]
@tyndria are you talking about PR #20 ? I can't see any tile redesign there

**Comment by tyndria** [comment]
@UladBohdan , Vika has deployed her changes, but now there are overwritten :(

File: src/tiles/tile.scss (hunk 2 above)
Comment (tyndria) [reply src/tiles/tile.scss]: .card-content.author.img class and .card-footer.author.img class has the same properties inside; the same for the class .name

File: src/tiles/tile.ejs (hunk 1 above)
Comment (tyndria) [reply src/tiles/tile.ejs]: Yes, it's better. But probably there is already a class from bulma, like 'is-flex', something like that, you could check. In this case you don't need to create your own class

Comment (UladBohdan) [reply src/tiles/tile.ejs]: bulma has `is-flex` indeed! thanks

File: src/tiles/tile.scss (hunk 2 above)
Comment (UladBohdan) [reply src/tiles/tile.scss]: sorry, my bad! I don't use `.card-footer` at all, that was experimental, forgot to remove it

**Comment by UladBohdan** [comment]
@tyndria I still can't see any Vika's tile-related changes, Vika hasn't pushed tile changes to `master`.
@andemerie have you?

**Comment by UladBohdan** [comment]
I've redesigned our navbar:

screencapture-localhost-3001-main-page-main-page-html-1516394076553

What do you think?

I've deployed changes on gh pages, you can try to hover sections / to check language dropdown.
https://babajka.github.io/babajka-markup/main-page/main-page.html

**Comment by Drapegnik** [comment]
@UladBohdan, I like this navbar!
About new footer:
screen shot 2018-01-20 at 16 10 12
a lot of free space & not symmetric alignmenttlooks ugly on desktops

@tyndria:
screen shot 2018-01-20 at 16 16 07
I think we should reduce this white "border" around icon

**Comment by UladBohdan** [comment]
I've pushed some footer redesign changes:
screencapture-localhost-3001-main-page-main-page-html-1516460254028

**Comment by UladBohdan** [comment]
Responsiveness for navbar and for tiles!


-----

-----

**Comment by UladBohdan** [comment]
@babajka/frontend can somebody check my css?

**Pull request merged**